cogs/induct.py

```diff
@@ -245,7 +244,7 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
         if applicant_role and applicant_role in induction_member.roles:
             await induction_member.remove_roles(
                 applicant_role,
-                reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
+                reason=f'{ctx.user} used TeX Bot slash-command: "/induct"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes